.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+
+on:
+  push:
+
+  # Allows manual bumps by API trigger
```

Comment:
Do we need this? Manually editing the GO_VERSION file should be doable, and this would simplify this action and provide only one way to update to a specific version (setting the GO_VERSION via git)